scripts/sync_prod

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+#
+# sync_prod - 0.1
+#
+#
+# sync_prod is a utility to sync database dumps from production and and to
+# sync media files (if applicable) for a Django application that are
+# user managed (i.e. uploaded photos for a CMS). Assumes that a proper
+# .my.cnf with credentials exists on target hosts for mysqldump.
+#
+# Dependencies:
+# - jq (system install via package manager for JSON parsing)
```

Comment:
Not really, at least not for bash and not for this sort of object-y way of configuring. (There's an exact equivalent for YAML called `yq`). Bash is old and fairly dumb. My rationale here is that it's one, relatively robust system dependency rather than maintaining a Python venv.

In theory, I could just specify parallel arrays in bash but that seemed *very* ugly and much easier to mess up, whereas we all know what a JSON object looks like.

Reply:
I think I had this config file format in mind: https://docs.python.org/3.4/library/configparser.html

I think the config file should be easy to read and edit; I think a plain text file would be easier than json (which  yes, we all know what it looks like, but it can be easy to make it invalid if you're editing manually).

If bash doesn't have an easy config file format is it still the right tool for this job?